README.md

```diff
@@ -8,101 +8,79 @@ This is a [level 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/mast
 
-We are going to continue making functionality that works with this data, such as finding the duration of a specific trip or the total amount of money a passenger has spent, and also make functionality to create a new trip.
+We are going to continue making functionality that works with this data, such as determining the duration of a specific trip and the total amount of money a user has spent, as well as the amount of money a driver has made and also add functionality to create a new trip and assign available drivers.
 
 ### The Code So Far
-#### Driver
-Each `Driver` has:
 
-**Attribute**|**Description**
------|-----
-id|The Driver's ID number
-name|The name of the Driver
-vehicle\_identification|The driver's Vehicle Identification Number (VIN Number), Each vehicle identification number should be a specific length to ensure it is a valid vehicle identification number
-trips | A list of trips the driver has gone on.
-status|Indicating availability, a driver's availability should be either `:AVAILABLE` or `:UNAVAILABLE`
+#### User
+A `User` represents a person with an account in our Rideshare service.  `Users` can use the service to take trips and can become Drivers with the service.
 
-Each `Driver` instance is able to:
-
-**Method**|**Description**
------|-----
-average_rating|retrieve an average rating for that driver based on all trips taken
-
-#### Passenger
-Each `Passenger` has:
+Each `User` has:
 
 **Attribute**|**Description**
 -----|-----
-id|The Passenger's ID number
-name|The name of the Passenger
-Phone Number|The Passenger's Phone Number which must be in phone number format (XXX) XXX-XXXX
-trips|A list of trips that only this passenger has taken
-
-Each `Passenger` instance is able to:
-
-**Method**|**Description**
------|-----
-get_drivers|retrieve the list of all previous driver instances associated with trips this passenger has taken
+id|The User's ID number
+name|The name of the User
+Phone Number|The User's Phone Number which must be in phone number format (XXX) XXX-XXXX
+trips|A list of trips that only this User has taken as a passenger
 
 #### Trip
+A `Trip` represents a journey a `User` has taken with the service.
+
 Each `Trip` has:
 
 **Attribute**|**Description**
 -----|-----
 id|The Trip's ID number
-passenger|The passenger on the trip
-driver|The driver for the trip
+User|The User on the trip
 start_time|When did this trip begin?
 end_time|When did this trip finish?
-rating|The rating given by the passenger, a number 1-5
+rating|The rating given by the User, a number 1-5
 cost|How much did the passenger pay?
 
 Each `Trip` instance is able to:
 
 **Method**|**Description**
 -----|-----
-driver|retrieve the associated driver instance
-passenger|retrieve the associated passenger instance
+User|retrieve the associated User instance
```

Comment:
I recognize that this table existed before, but this is confusing to me. Why is `user` a method (presumably a reader) but the other attributes are not?